_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -842,130 +857,683 @@ If you need to brush up on how the IaC Library works, read the
+  description = "The names of the configuration recorder."
+  value       = module.root_baseline.config_recorder_names
+}
+
+# ---------------------------------------------------------------------------------------------------------------------
+# ORGANIZATIONS OUTPUTS
+# ---------------------------------------------------------------------------------------------------------------------
+
+output "organization_arn" {
+  description = "ARN of the organization."
+  value       = module.root_baseline.organization_arn
+}
+
+output "organization_id" {
+  description = "Identifier of the organization."
+  value       = module.root_baseline.organization_id
+}
+
+output "master_account_arn" {
+  description = "ARN of the master account."
+  value       = module.root_baseline.master_account_arn
+}
+
+output "master_account_id" {
+  description = "Identifier of the master account."
+  value       = module.root_baseline.master_account_id
+}
+
+output "master_account_email" {
+  description = "Email address of the master account."
+  value       = module.root_baseline.master_account_email
+}
+
+output "child_accounts" {
+  description = "A map of all accounts created by this module (NOT including the root account). The keys are the names of the accounts and the values are the attributes for the account as defined in the aws_organizations_account resource."
+  value       = module.root_baseline.child_accounts
+}
+
+# ---------------------------------------------------------------------------------------------------------------------
+# CLOUDTRAIL OUTPUTS
+# ---------------------------------------------------------------------------------------------------------------------
 
-[[merge_release_password_policy_module]]
-===== Merge and release your password policy module
+output "cloudtrail_trail_arn" {
+  description = "The ARN of the cloudtrail trail."
+  value       = module.root_baseline.cloudtrail_trail_arn
+}
 
-Once the module is working the way you want, submit a pull request, get your changes merged into the `master` branch,
-and create a new versioned release by using a Git tag. For example, to create a `v0.3.0` release:
+output "cloudtrail_s3_bucket_name" {
+  description = "The name of the S3 bucket where cloudtrail logs are delivered."
+  value       = module.root_baseline.cloudtrail_s3_bucket_name
+}
+
+output "cloudtrail_s3_bucket_arn" {
+  description = "The ARN of the S3 bucket where cloudtrail logs are delivered."
+  value       = module.root_baseline.cloudtrail_s3_bucket_arn
+}
+//todo -> same as last
+//output "cloudtrail_kms_key_arn" {
+//  description = "The ARN of the KMS key used by the S3 bucket to encrypt cloudtrail logs."
+//  value       = module.root_baseline.cloudtrail_kms_key_arn
+//}
+//
+//output "cloudtrail_kms_key_alias_name" {
+//  description = "The alias of the KMS key used by the S3 bucket to encrypt cloudtrail logs."
+//  value       = module.root_baseline.cloudtrail_kms_key_alias_name
+//}
```

Comment:
same here: 

These are currently not recognised & thus is invalid tf. This has been taken from cis-repo/examples/landingzone/account-baseline-root